RobotArmAPP/Classes/WiFiAPConnection.cs

```diff
@@ -0,0 +1,141 @@
+﻿using System;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Windows.Devices.WiFi;
+using Windows.Security.Credentials;
+using Windows.UI.Popups;
+
+namespace RobotArmAPP
+{
+    class WiFiAPConnection
+    {
+        public static string SSID { get; set; } = "robotarm";
+        public static string PW { get; set; } = "0xcrossbots";
+
+        private int status = 0; //0 = notConnected, 1 = isConnected, 2 = isConnecting
+
+        WiFiAdapter wifiAdapter;
+
+        public async void RequestWifiAcess()
+        {
+            try
+            {
+                var access = await WiFiAdapter.RequestAccessAsync();
+                if (access == WiFiAccessStatus.Allowed)
+                {
+                    var result = await Windows.Devices.Enumeration.DeviceInformation.FindAllAsync(WiFiAdapter.GetDeviceSelector());
+                    if (result.Count >= 1)
+                    {
+                        wifiAdapter = await WiFiAdapter.FromIdAsync(result[0].Id);
+                    }
+                }
+            }
+            catch { }
+        }  //Method that needs to be called in the Page_Loaded or OnNavigatedTo function to request access to the wifi controls
+
+        private async Task<string> VerifyAP(int appConnectionOK)
+        {
+            HttpClient cliente = new HttpClient();
+            HttpResponseMessage response = new HttpResponseMessage();
+            string resultado = await cliente.GetStringAsync("http://10.10.10.10/?param=" + appConnectionOK);
```

Comment:
```suggestion
            string resultado = await cliente.GetStringAsync(baseUrl + "?param=" + appConnectionOK);
```
Dê uma olhada em arquivos `.env` para guardar esse tipo de url enquanto não tem nenhuma página de configuração

Reply:
Eu criei uma variável, você não recomenda ?
public static string baseUrl { get; set; } = "http://10.10.10.10";
public static string portForSendFrames { get; set; } = ":18";
public static string portForVerifyConnection { get; set; } = null;